.github/workflows/ci2.yml

```diff
@@ -74,12 +74,12 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        distro: [rocky, fedora, leap.15]
```

Comment:
This is currently marked as required for "rocky" so changing to rocky.9 will mean changing the GitHub configuration, we could either do that or I could leave this at rocky without the .9.  I'm happy either way.

Reply:
I did the latter.